src/components/Header/Header.module.scss

```diff
@@ -1,25 +1,25 @@
-.header {
-    background: #c4c4c4;
-    border-bottom: 0.3px solid #000000;
-}
-
-.headerButton {
-    padding: 0;
-    margin: 6px 0 7px 18px;
-    border: none;
-    border-radius: 50%;
-
-    &:hover {
-        border: 1px solid #000000;
-        box-sizing: border-box;
-        margin: 5px 0 6px 17px;
-    }
-}
-
-.headerAvatar {
-    width: 47px;
-    height: 43px;
-    margin: 0;
-    padding: 0;
-    background-image: url("../../images/user-avatar.svg");
-}
+.header {
+    background: #c4c4c4;
+    border-bottom: 0.3px solid #000000;
+}
+
+.headerButton {
+    padding: 0;
+    margin: 6px 0 7px 18px;
+    border: none;
+    border-radius: 50%;
+
+    &:hover {
+        border: 1px solid #000000;
+        box-sizing: border-box;

```

Comment:
I think if you remove "border-box" here and above, you will not need to adjust margins when adding border